### PR TITLE
Feat/profile page

### LINKED ISSUE #3: US3 - Dog Profile Page
## Value proposition

As an user
I want to see a new page, when I click on one of the list items
In order to see more informations suitable for the breed

## Description

Bildschirmfoto 2023-08-14 um 08 59 37

### On the Overview Page
- [x] the list items are clickable
- [x] when clicked, a new page open

### On the Profile Page
- [x] at the top of the page there is a button to get back to the overview
- [x] I can see the informations I inserted for this profile (name, age and breed)
- [x] I can see information that fits the breed

## Tasks

- [x] `create a branch feat/profile-page`

### For the Overview Page
- [x] the list items are clickable
- [x] add a hover state

### For the Profile Page
- [x] create a dynamic route for the dog profiles
useRouter()
- [x] display a button at the top "back to Overview"
- [x] display information from the state
- [x] display information from the API depending on the inserted breed

## Size - S to M
I expect to take 2 to 4 hours to complete

### PR DESCRIPTION
**Pull request by ninaassmann**
- pets are clickable
- add profile page -> [slug].js
- used dynamic routing
- add suitable information about the pet breed

### CODE REVIEW

**Comment by ninaassmann** [comment]
vercel didn't update after my last push
use this link for testing

https://nf-capstone-project-5isg7wbg4-ninaassmann.vercel.app/

File: components/ListItem.js [hunk 1]
```diff
@@ -1,25 +1,36 @@
...
-  const age = `${years}.${months} years`;
+export default function ListItem({
+  id,
+  name,
+  breed,
+  birthday,
+  slug,
+  calculateAge,
+}) {
+  const age = calculateAge(birthday);
```
Comment (F-Kirchhoff) [inline components/ListItem.js:6]: does this function needs to be a prop or can you put it in a utils folder and simply import the function in this file?

**Review by F-Kirchhoff — COMMENTED** [review]
LGTM except one question. Please see below.

File: components/ListItem.js (hunk 1 above)
Comment (ninaassmann) [reply components/ListItem.js:6]: thx

**Review by F-Kirchhoff — APPROVED** [review]
LGTM

**Pull request merged**